api/src/controllers/activity.ts

```diff
@@ -68,54 +66,7 @@ router.get(
 	respond,
 );
 
-const createCommentSchema = Joi.object({
-	comment: Joi.string().required(),
-	collection: Joi.string().required(),
-	item: [Joi.number().required(), Joi.string().required()],
-});
-
-router.post(
-	'/comment',
-	asyncHandler(async (req, res, next) => {
-		const service = new ActivityService({
-			accountability: req.accountability,
-			schema: req.schema,
-		});
```

Comment:
Per the other comments, we seem to want backwards compatibility. Shouldnt we then not just use this existing activity controller to actually call the methods on the new `CommentsService`?

The migration moves everything to the comments table, why are we then still patching and deleting on the activity table instead of the comments table, the data is there? I'd say if someone then POSTs a new comment on `/activity/comment`, we ought to simply call `CommentsService.createOne` and thats it. The comments table should be the new source of truth right?

Then we also dont need two requests in `comments-sidebar-detail.vue`, we only need to fetch from the comments table. Pls correct me if I misunderstand something :pray:

Reply:
Great points! Yes to the above questions. The `CommentsService` is being refactored 👍

Reply:
I believe this originally done for the edge case where the migration was running and a comment was created.

Since the migration was refactored to only create the table this is most likely no longer a concern.